tests/test_soup_searcher.py

```diff
@@ -0,0 +1,29 @@
+import pytest
+from amz_review_scraper.soup_searcher import find_attribute
+import codecs
+import bs4
+import os
+
+my_path = os.path.abspath(os.path.dirname(__file__))
+path = os.path.join(my_path, "materials/output_file.html")
+
+raw_html = codecs.open(path,'r')
+soup = bs4.BeautifulSoup(raw_html, "lxml")
```

Comment:
This is a pretty good approach to test this for now.
Although, it might be nice to create a smaller HTML chunk to use in tests, so it is easier to grok the test input.
But since this works right now, I might get to that later.